deeprank/generate/GridTools.py

```diff
@@ -554,7 +546,7 @@ def map_features(self, featlist, transform=None):
                     dict_data[feature_name + '_chainB'] = np.zeros(self.npts)
                 else:
                     dict_data[feature_name] = np.zeros(self.npts)
-            else:
+            else: # do we need that ?!
```

Comment:
It's never used before and we never tried `transform` neither. It's fine to keep it.